pow-migration/src/history/mod.rs

```diff
@@ -196,7 +196,7 @@ pub async fn migrate_history(
             let mut txn = env.write_transaction();
             history_store.add_to_history(
                 &mut txn,
-                0,
+                block_height,
```

Comment:
I think this won't work. For this the PoW genesis block must be known and this is not the case for the history migration